README.md

```diff
@@ -1,220 +1,680 @@
- - disable RT throttling
- - isolate ethercat task and network IRQ on a dedicated core
- - change network IRQ priority
-
-#### PikeOS
- - Tested on PikeOS 5.1 for native personnality (p4ext)
- - You have to provide the CMake cross-toolchain file which shall define PIKEOS variable (or adapt the main CMakelists.txt to your needs)
- - examples are provided with a working but non-optimal process/thread configuration (examples/PikeOS/p4ext_config.c): feel free to adapt it to your needs
-
-#### Windows
-Absolutely NOT suitable for real time use, but it can come in handy to run tools that rely on EtherCAT communication.
-
-To build the library, you need to install:
-- conan for windows (tested with 2.9.1)
-- gcc for Windows (tested with w64devkit 2.0.0)
-- npcap (tested with driver 1.80 + SDK 1.70 through conan package)
-
-
-### Build:
-#### Easy setup (Linux only)
-  1. Install conan package manager. For instance with uv package manager:
-  ```
-  uv venv
-  source .venv/bin/activate
-  uv pip install conan
-  ```
-
-  2. Launch the script `setup_build.sh path_to_build_folder`.
-  3. Configure CMake project and build:
-  ```
-  cd path_to_build_folder
-  cmake .. -DCMAKE_BUILD_TYPE=Release
-  make
-  ```
-
-#### Manual setup
-KickCAT project is handled through CMake. To build the project, call CMake to configure it and then the build tool (default on Linux is make):
-  1. Create the build folder
-  ```
-  mkdir -P build
-  ```
-  2. Install conan and the dependencies with it (you may need to adapt the profile to suit your current installation). This step is mandatory for Windows, optional otherwise
-  ```
-  python3 -m venv kickcat_venv
-  source kickcat_venv/bin/activate
-  pip install conan
-
-  conan install conan/conanfile_linux.txt -of=build/ -pr:h conan/your_profile_host.txt -pr:b conan/your_profile_target.txt --build=missing -s build_type=Release
-  ```
-
-  2. Configure the project (more information on https://cmake.org/cmake/help/latest/)
-  ```
-  cd build
-  cmake .. -DCMAKE_BUILD_TYPE=Release
-  ```
-  3. Build the project
-  ```
-  make
-  ```
-
-### Build unit tests (optional)
-Select the BUILD_UNIT_TEST option in CMake, either through ccmake or by calling cmake tool in your build folder like this:
+# KickCAT
+
+<p align="center">
+  <strong>A lightweight, efficient EtherCAT master/slave stack for embedded systems</strong>
+</p>
+
+<p align="center">
+  <em>Kick-start your slaves! ⚡</em>
+</p>
+
+---
+
+## Overview
+
+KickCAT is a thin EtherCAT stack designed to be embedded in complex software with efficiency in mind. It provides both **master** and **slave** implementations, supporting multiple operating systems and hardware platforms.
+
+**Key Features:**
+- Works with Linux (including RT-PREEMPT), Windows, and PikeOS
+- Full state machine support (INIT → PRE-OP → SAFE-OP → OP)
+- CoE (CANopen over EtherCAT) support with SDO read/write
+- Interface redundancy
+- Bus diagnostics and error handling
+- Python bindings available
```

Comment:
remove available, seems redundant with the list

Maybe something like that:
Python bindings for master side